src/translations/dictionary.ts

```diff
@@ -24,6 +24,42 @@ const dictionary = {
     m: _('m', 'm', 'm'),
   },
   a11yRouteTimePrefix: _('rutetid ', 'route time ', `rutetid `),
+  missingRealTimePrefix: _('ca. ', 'ca. ', `ca. `),
```

Comment:
Should this be "approx." in English? 🤔

Reply:
Apparently "ca." is used in the English language as an abbreviation for "circa", thus can be used in this instance.